scripts/globals/moghouse.lua

```diff
@@ -239,3 +268,76 @@ xi.moghouse.addMogLockerExpiryTime = function(player, numBronze)
 
     return true
 end
+
+xi.moghouse.exitJobChange = function(player, prevZone)
+    if player:getCharVar('[Moghouse]Exit_Pending') == 1 and prevZone == player:getZoneID() then
+        player:setVolatileCharVar('[Moghouse]Exit_Pending', 0)
```

Comment:
`[Moghouse]Exit_Pending` set in both sides of this if

Reply:
Fixed